uom_alias/models/uom_uom.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2025  Renato Lima - Akretion
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, fields, models
+from odoo.osv import expression
+
+
+class Uom(models.Model):
+    _inherit = "uom.uom"
+
+    alias_ids = fields.One2many(
+        comodel_name="uom.alias",
+        inverse_name="uom_id",
+        string="Uom Alias",
+    )
+
+    @api.model
+    def _name_search(
+        self, name="", args=None, operator="ilike", limit=100, name_get_uid=None
+    ):
+        args = args or []
+        domain = []
+        if name:
+            domain = ["|", ("name", operator, name), ("alias_ids.code", operator, name)]
+        return self._search(
+            expression.AND([domain, args]),
+            limit=limit,
+            access_rights_uid=name_get_uid,
+        )
```

Comment:
```suggestion
    @property
    def _rec_names_search(self):
        return list(set(super()._rec_names_search + ["alias_ids.code"]))
```

In Odoo 16+, I see many people implementing it this way. Could this be better?